language/Types.md

```diff
@@ -74,7 +74,16 @@ instance showPercentage :: Show Percentage where
 
 ## Functions
 
-Functions in PureScript are like their Javascript counterparts, but always have exactly one argument.
+Functions in PureScript are like their Javascript counterparts, but always have exactly one argument. Function declarations and lambdas (anonymous functions) with multiple arguments are shorthand for curried lambda abstractions. For example, the following are all equivalent:
+
+```purescript
+add x y = x + y
+add = \x y -> x + y
+add = \x -> \y -> x + y
+add = (+)
+```
+
+Despite this overwhelming convention, uncurried JavaScript functions are can be used through the [purescript-functions](https://pursuit.purescript.org/packages/purescript-functions/) library. For more information see [The Foreign Function Interface](http://www.purescript.org/learn/ffi/).
```

Comment:
Okay ... What about a very short note with two links? Or just revert the changes?

> Functions in PureScript (defined as `input -> output`) are like their JavaScript counterparts, but are curried to always have exactly one argument. Currying works just like [Haskell](https://wiki.haskell.org/Currying).
>
> Uncurried JavaScript functions are also available through the [purescript-functions](https://pursuit.purescript.org/packages/purescript-functions/) library.

The reason I include the link to Haskell is that it explains several features of currying quite well: function returns function, partial application, associativity, pointfree, etc.